.env.example

```
@@ -26,19 +33,19 @@ REDIS_PORT=6379
 # Portal: https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps
 MSGRAPH_CLIENT_ID=your-azure-client-id-here
 MSGRAPH_TENANT_ID=your-azure-tenant-id-here
-MSGRAPH_CLIENT_SECRET=your-azure-client-secret-here
+MSGRAPH_CLIENT_SECRET=your_msgraph_client_secret_here  # Run: node scripts/generate-secure-secrets.js msgraph
 
 # Webhook Configuration
 WEBHOOK_URL=https://your-domain.com/api/webhooks/microsoft-graph
-WEBHOOK_CLIENT_STATE=your-secret-client-state-here
+WEBHOOK_CLIENT_STATE=your_webhook_client_state_here  # Run: node scripts/generate-secure-secrets.js webhook
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The WEBHOOK_CLIENT_STATE key should go before the WEBHOOK_URL key

(UnorderedKey)